src/test/java/uk/gov/dhsc/htbhf/steps/ClaimValuesTestDataFactory.java

```diff
@@ -8,7 +8,7 @@
  */
 public class ClaimValuesTestDataFactory {
 
-    public static ClaimValues buildDefaultClaimValues() {
+    public static ClaimValues buildClaimValuesForANonPregnantWoman() {
```

Comment:
Does she have children?

Reply:
Although, looking at the other methods maybe it doesn't matter

Reply:
Whether the claimant has children or not isn't a part of ClaimValues, so its not relevant here yet. For reference, the default navigation does add a child's date of birth, but this isn't stored in ClaimValues. I do plan to move the children dobs into the ClaimValues at some point, but not yet.